compiler/src/dotty/tools/dotc/transform/TreeMapWithStages.scala

```diff
@@ -149,6 +149,11 @@ abstract class TreeMapWithStages(@constructorOnly ictx: Context) extends TreeMap
         case (_:Import | _:Export) =>
           tree
 
+        case _: Template =>
+          for sym <- tree.symbol.owner.info.decls do
+            markSymbol(sym)
+          super.transform(tree)
```

Comment:
Why isn't mapOverTree used like in the other cases where we call markSymbol?

Reply:
No reason. I updated it to use `mapOverTree`.